doc/api/url.markdown

```diff
@@ -64,6 +64,11 @@ string will not be in the parsed object. Examples are shown for the URL
 
     Example: `'#hash'`
 
+The following characters and spaces (`' '`) will be automatically escaped in the
```

Comment:
This reads a bit funny to me, maybe it should be reworded to something like:
<pre>
Spaces (`' '`) and the following characters will be ....
</pre>